easy_access/enrichment/osiris.py

```
@@ -241,21 +260,40 @@ async def fetch_single_course(course_code: int) -> None:
+                            async with get_session_factory()() as session:
+                                # Check if exists
+                                result = await session.execute(
+                                    select(MissingCourse).where(
+                                        MissingCourse.cursuscode == course_code
+                                    )
+                                )
+                                existing = result.scalar_one_or_none()
+                                if existing:
+                                    # Update modified_at
+                                    await session.execute(
+                                        update(MissingCourse)
```

Ruff (0.14.6)

Undefined name `update`

(F821)